### DISCUSSION TITLE
Config option for command aliases and more specific permissions for /claimexplosions

### DISCUSSION THREAD

**Discussion by Hydroxycobalamin** [GriefPrevention/GriefPrevention#1058] [Ideas]
**As** a server owner
**I want** to clean up the command suggestions list
**Because** I'm not a fan of millions of commands in the suggestion list

I'd like to clean up the command suggestion list from some commands and GriefPrevention add a lot of aliases. 
The ability to remove or change them in config would be great!

Also a separate permission for /claimexplosions would be nice too.

Optional questions:

**Is your feature request related to a problem? Please describe.**
No.

**Describe alternatives you've considered**
Blocking them via dScript.

**Comment by RoboMWM** [comment]
Use the Bukkit-included commands.yml to set aliases for whatever you want, or to remove them. (Unsure if it removes from the tabcomplete list but that'd be outside the realm of plugin.yml.)
